### ISSUE TITLE
Uncaught Error: `initialFocus` refers to no known node

### ISSUE THREAD

**Issue by SonyaOrlova** [davidtheclark/focus-trap#81]
Hi! Case for React App:
```
state = { isLoading: true };

componentDidMount() {
   if (// some logic) this.setState({ isLoading: false });
}

render () {
   if (this.state.isLoading) {
      return <React-aria-modal-first/>
   } else {
      return <React-aria-modal-second/>
}
```

So i have two react-aria-modals, and on the first Render i always show `<React-aria-modal-first/> `and on the second Render (if condition's result in componentDidMount is true) i need to show `<React-aria-modal-second/>`, but due to delay before the element is focused (https://github.com/davidtheclark/focus-trap/blob/master/index.js#L141) i always get "Uncaught Error: `initialFocus` refers to no known node"  because second render in React happens faster (first modal has been already removed from the DOM) than focus-function on first modal is executed. But in this case i don't want to see this error because there is no problems in my app or React

**Comment by davidtheclark** [comment]
Hi @SonyaOrlova. What would you think of rendering *no* modal until you know which one to render (in `componentDidMount`), then rendering the one that will stick around?

**Comment by SonyaOrlova** [comment]
unfortunately in our project there is no way to do that correctly because the view consists only of modals, we can just render _null_ in componentDidMount, but it is not an appropriate solution anyway
(looks like crutch)

**Comment by davidtheclark** [comment]
Fix published in 5.0.1.

**Comment by SonyaOrlova** [comment]
could you please update focus-trap version in dependencies  in focus-trap-react https://github.com/davidtheclark/focus-trap-react/blob/master/package.json ?
